frontend/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM node:12
+
+# Setting working directory. All the path will be relative to WORKDIR
+WORKDIR /usr/src/app
+
+# Installing dependencies
+COPY package*.json ./
+RUN npm install
```

We're using `yarn` and we have `yarn.lock` file. Let's use that here, too.
```suggestion
COPY package.json yarn.lock ./
RUN yarn install --frozen-lockfile
```

Same further down in the file.